pkgs/development/libraries/shibboleth-sp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, autoreconfHook, boost, fcgi, openssl, opensaml-cpp, log4shib, pkgconfig, xercesc, xml-security-c, xml-tooling-c }:
+
+stdenv.mkDerivation rec {
+  name = "shibboleth-sp-${version}";
+  version = "2.6.0";
+
+  src = fetchgit {
+    url = "https://git.shibboleth.net/git/cpp-sp.git";
+    rev = "9ebba5c3a16d03769f436e383e4c4cdaa33f5509";
+    sha256 = "1b5r4nd098lnjwr2g13f04ycqv5fvbrhpwg6fsdk8xy9cigvfzxj";
+  };
+
+  # Needs pkgconfig to find systemd
```

Comment:
Oh, it probably uses `pkgconfig` and then invokes it and realizes that `systemd` isn't there and omits `systemd` support in that case. Comment is slightly confusing but not the end of the world. I'd change it if you end up changing this commit for other reasons but not bother otherwise.